internal/guestrequest/types.go

```diff
@@ -66,18 +67,24 @@ type LCOWNetworkAdapter struct {
 	EncapOverhead   uint16 `json:",omitempty"`
 }
 
+type LCOWContainerConstraints struct {
+	Windows specs.WindowsResources `json:",omitempty"`
```

Comment:
Would we utilize the Windows fields in the future? I see the only use of this at the moment (https://github.com/microsoft/hcsshim/pull/931/files#diff-04078e400c5f445fb34c4927b97930a0d1dbd00005819d7b7a77022d092f365cR946) just casts into `LinuxResources` and plops it into the request.

Reply:
We don't currently use it. The idea was that if we *needed* to we could send something other than just the linux resources to LCOW, but I think we could probably just take this out for now.

Reply:
Sounds good to me :)

Reply:
Actually, if I do that I'll need to update opengcs too. Are you okay with this staying in?

Reply:
Sure that's fine, maybe just make a note about this is here for future use cases or something of the sort.